Gopkg.toml

```diff
@@ -0,0 +1,61 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+
+[[constraint]]
+  name = "github.com/inconshreveable/log15"
+  version = "2.11.0"
+
+[[constraint]]
+  name = "github.com/jessevdk/go-flags"
+  version = "1.2.0"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/satori/go.uuid"
```

Comment:
The previous `glide.yaml` file had lots of revisions fixed. Are we moving from the previously conservative way of fixing almost all versions now that `vendor` is in the repo?

Reply:
The fixed version aren't required with go dep, it calculates the latest compatible library, also the glide was migrated